### ISSUE TITLE
dunstctl script doesn't work anymore

### ISSUE THREAD

**Issue by gfarrell** [dunst-project/dunst#748]
I'm trying to write a script to show a do-not-disturb control / indicator in Polybar. I'd like to be able to access the dunst status, so tried using the `dunstctl` script. I'm new to things like DBus, and though I tried to search through the source code for dunst, I'm not entirely sure how to find what I'm looking for to fix this issue (which I'm happy to contribute a PR for if someone points me in the right direction).

It seems that the dunstctl script doesn't know how to talk to dunst over dbus. I'm slightly confused by this as the script is newer than my dunst version (1.4.1). The problem seems to be in the line which defines:

```shell
 dbus_ifac="org.dunstproject.cmd0" 
```

As DBus responds saying this is not a known interface.

### Steps to reproduce

1. Download and save dunstctl script (can also just reproduce what it's doing but it's easier to use the script):

```shell
curl https://raw.githubusercontent.com/dunst-project/dunst/master/dunstctl -o ./dunstctl
chmod +x ./dunstctl
```

2. Check dunst is working

```shell
notify-send "Hello" # should see a notification pop up
```

3. Try using `dunstctl is-paused`

```shell
~ > ./dunstctl is-paused
Error org.freedesktop.DBus.Error.InvalidArgs: No such interface “org.dunstproject.cmd0”
Failed to communicate with dunst, is it running? Or maybe the version is outdated. You can try 'dunstctl debug' as a next debugging step.
```

4. Try using `dunstctl debug`

```shell
~ > ./dunstctl debug
dunst version: 1.4.1
Dunst controlling interface not available. Is the version too old?
```

### Installation info

- Version: `Dunst - A customizable and lightweight notification-daemon 1.4.1 (2019-07-03)`
- Install type: `sudo pacman -S dunst`
- Distro and version: `Linux anaximander 5.7.7-arch1-1 x86_64 GNU/Linux `

**Comment by tsipinakis** [comment]
> I'm slightly confused by this as the script is newer than my dunst version (1.4.1)
> [..]
> Dunst controlling interface not available. Is the versio

Well, that's the problem :). The dbus interface used for `dunstctl` was first implemented in `1.5`. In general, the script should be the same version as dunst as the interface can change between versions.

**Comment by gfarrell** [comment]
Ah I misread some of the versioning, and thought that dunstctl had been added to be compatible with 1.4.

Makes sense, I’ll see what’s going on in the arch repo so I can upgrade.
